Assets/Scripts/StandardSamples/UI/Common/UIDebugLog.cs

```diff
@@ -341,9 +350,12 @@ private void Update()
             }
 
             // FPS
-            deltaTime_FPS += (Time.deltaTime - deltaTime_FPS) * 0.1f;
-            float fps = 1.0f / deltaTime_FPS;
-            FPSValue.text = Mathf.Ceil(fps).ToString();
+            if (FPSValue != null)
+            {
+                deltaTime_FPS += (Time.deltaTime - deltaTime_FPS) * 0.1f;
+                float fps = 1.0f / deltaTime_FPS;
+                FPSValue.text = Mathf.Ceil(fps).ToString();
+            }
```

Comment:
Would be good to check to prevent accidental divide-by-zero. I would favor this version personally:
```csharp
if (FPSValue != null)
{
    deltaTime_FPS += (Time.deltaTime - deltaTime_FPS) * 0.1f;
    float fps = (deltaTime_FPS != 0) ? 1.0f / deltaTime_FPS : float.PositiveInfinity;
    FPSValue.text = Mathf.Ceil(fps).ToString();
}
```
A clearer implementation that avoids the inclusion of a ternary statement would be:
```csharp
if (FPSValue != null)
{
    deltaTime_FPS += (Time.deltaTime - deltaTime_FPS) * 0.1f;
    
    // Prevent division by zero by checking if deltaTime_FPS is not zero
    if (deltaTime_FPS != 0)
    {
        float fps = 1.0f / deltaTime_FPS;
        FPSValue.text = Mathf.Ceil(fps).ToString();
    }
    else
    {
        FPSValue.text = "Inf"; // Or handle the case as needed
    }
}
```

_Either way_, what is the purpose of multiplying `(Time.deltaTime - deltaTime_FPS)` by `0.1f`?

Reply:
Resolved as recommended, though I'm using "NaN" instead of infinite.

I have.. no idea why this is multiplied by .1f. Honestly this bit of code doesn't make sense to me.